.github/workflows/mac-artifact.yml

```
@@ -2,6 +2,8 @@ name: Mac Artifact
 on:
   push:
     branches: [ master, test-ready ]
```

```suggestion
    branches: [ master, test-ready, "bugfix-*", actionsHub ]
```
I find both these useful.  The bugfix one is necessary I think